src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplyForm.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
```

Comment:
This currently breaks the `yarn flow check`, I see I need to extend the https://github.com/jankowiakdawid/box-ui-elements/blob/annotationThread-handleCreateReply/flow-typed/npm/react-intl_v2.x.x.js#L1 to fix it

Reply:
There is a script that checks for `useIntl` being used, I am unsure why exactly but from what I understand we should be using the `intl` from via `injectIntl` to formatMessage e.g. https://github.com/box/box-ui-elements/blob/830bf71bef5720a5e6ba128afa35152112674c17/src/elements/content-explorer/ShareDialog.js#L54

Reply:
Ok, then as @jstoffan suggested I'll use `injectIntl`